pymoliere/construct/embedding_util.py

```diff
@@ -1,25 +1,25 @@
-from typing import Tuple, Iterable, ClassVar
```

Comment:
These changes are simply because I sorted the imports. Arguably I shouldn't have done that in this commit.